wordpress-mobile/WordPress-Android
# Update stats feature announcement logic

This update the logic of stats feature announcement.

**Changes:**
- Shows on launch if the user has a site, and their site is a wpcom / jetpack site?
- The dismiss button will permanently dismiss the alert
- The Remind Me button will re-show the alert on app launch
- The bulb icon is changed to stats icon

To test:

Setup:

- Launch Jetpack app.
- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings).
- Select Debug Settings.
- Find StatsRevampV2FeatureConfig under Features in development enable it and restart app.
- Launch / Re-Launch Jetpack app.

Test:

- Notice the new Feature announcement dialog pops up.
- Tap on [Try it now] and ensure it opens Stats.
- Relaunch or Return Feature announcement.
- Tap on [Remind me later] and it should dismiss the dialog.
- Relaunch the app.
- Ensure the announcement screen appears again.
- Tap on X on top right hand corner and it should dismiss the dialog.
- Relaunch the app.
- Ensure the announcement screen don't appear.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
